driver-core/src/main/com/mongodb/client/model/changestream/ChangeStreamDocument.java

```diff
@@ -89,7 +93,8 @@ public ChangeStreamDocument(@BsonProperty("operationType") final String operatio
             @Nullable @BsonProperty("updateDescription") final UpdateDescription updateDescription,
             @Nullable @BsonProperty("txnNumber") final BsonInt64 txnNumber,
             @Nullable @BsonProperty("lsid") final BsonDocument lsid,
-            @Nullable @BsonProperty("wallTime") final BsonDateTime wallTime) {
+            @Nullable @BsonProperty("wallTime") final BsonDateTime wallTime,
+            @Nullable @BsonProperty final BsonDocument extraElements) {
```

Comment:
I was a bit surprised that this worked to combine this annotation with no name here with the one on the field, but it does.

Reply:
I'm not sure.  I just know that every parameter to the `BsonCreator`-annotated constructor has to be annotated with `BsonProperty` or else it throws.